engine/src/main/java/org/terasology/rendering/logic/LightComponent.java

```diff
@@ -15,13 +15,14 @@
  */
 package org.terasology.rendering.logic;
 
-import org.terasology.reflection.metadata.FieldMetadata;
 import org.terasology.entitySystem.Component;
 import org.terasology.math.geom.Vector3f;
 import org.terasology.network.Replicate;
 import org.terasology.network.ReplicationCheck;
+import org.terasology.reflection.metadata.FieldMetadata;
 
 /**
+ * Add this component to an entity for it to transmit light from its location.  By default the component is configured to act similarly to a placed torch block.
  */
 // TODO: Split into multiple components? Point, Directional?
 public final class LightComponent implements Component, ReplicationCheck {
```

Comment:
Indeed it looks almost mirror-like in the handling of the reflected light. Personally I feel that the light source shouldn't have any shininess parameter at all: it should have an (RGB) intensity and a way for that intensity to falloff with distance, but it should be up to a surface's shader to make the surface look shiny or not.